pkg/autoscaler/metrics/collector_test.go

```diff
@@ -116,13 +117,92 @@ func TestMetricCollectorCRUD(t *testing.T) {
 	})
 }
 
+func TestMetricCollectorScraperMovingTime(t *testing.T) {
+	logger := TestLogger(t)
+
+	mtp := &fake.ManualTickProvider{
+		Channel: make(chan time.Time),
+	}
+	now := time.Now()
+	fc := fake.Clock{
+		FakeClock: clock.NewFakeClock(now),
+		TP:        mtp,
+	}
```

Comment:
Could we move this part to above where it is used? i.e. before `coll`.

Reply:
above or below?